pubnub-kotlin/pubnub-kotlin-impl/src/main/kotlin/com/pubnub/internal/v2/PNConfigurationImpl.kt

```diff
@@ -63,142 +63,142 @@ class PNConfigurationImpl(
         BasePNConfigurationImpl.Builder(defaultConfiguration),
         PNConfiguration.Builder,
         PNConfigurationOverride.Builder {
-            private val log = LoggerFactory.getLogger(this::class.simpleName)
+        private val log = LoggerFactory.getLogger(this::class.simpleName)
```

Comment:
Was previous formatting improper?

Reply:
yeah the auto formatter changed this for some reason

Reply:
Looks bit strange to me.

Reply:
Now I get it.